mscore/prefsdialog.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>706</width>
-    <height>486</height>
+    <width>904</width>
+    <height>579</height>
```

Comment:
Qt Creator "just did it", but I didn't change anything that caused the size to change. I think it happens because on macOS the controls might have other default sizes, so when loading the file, Qt Creator adjusts the size of the dialog to fit. And on runtime, the minimum size is different than in Qt Creator anyway, because of the style.

Reply:
So if anybody is doing a PR about prefsdialog.cpp, they can undo/modify this change if they want.